lib/graph3d/Graph3d.js

```diff
@@ -67,6 +67,29 @@ var DEFAULTS = {
 
   style            : Graph3d.STYLE.DOT,
   tooltip          : false,
+
+  tooltipStyle     : {
```

Comment:
I haven't tested it yet. Please make sure the existing tooltips are basically not changed, or we have to consider this as a breaking change.